compile.rb

```diff
@@ -62,6 +62,8 @@ def format_emojis(emojis)
 
 $stderr.print "Loading annotations"
 Dir[
+  "cldr/common/annotations/en.xml",
+  "cldr/common/annotationsDerived/*.xml",
```

Comment:
Why is the `annotationsDerived` repeated?

Reply:
I use rofi and rofimoji to display a menu that shows me the emojis from all_emojis.txt to put them on the clipboard. If I take the approach of not parsing "en" first, I get a Danish description text. English is better then. Possibly here is something I did not understand, since I know ruby only from this project.

Without this change:
![image](https://user-images.githubusercontent.com/79472/117615347-e36a2980-b169-11eb-83ed-f01a855b2008.png)

With this change:
![image](https://user-images.githubusercontent.com/79472/117615860-9e92c280-b16a-11eb-81c2-77670d10abe4.png)


Maybe more needs to be changed in compile.rb or other ruby files?

Reply:
`all_emojis.txt` was supposed to use English, so I should look into that.

However, I guessed this was the problem that caused you to load `en` first, but my question was for the one below.

As it is now you load:

* English
* Derived
* All languages including English
* Derived (again)

I was unsure why Derived were loaded again before the other languages. Maybe it's just a mistake? :slightly_smiling_face: 

I'll investigate why `all_emojis.txt` wasn't using the English names.